.github/actions/e2e/install-karpenter/action.yaml

```diff
@@ -58,10 +58,7 @@ runs:
 
       # Parse minor version to determine whether to enable the webhooks
       RELEASE_VERSION_MINOR="${K8S_VERSION#*.}"
-      WEBHOOK_ENABLED=false
-      if (( RELEASE_VRESION_MINOR < 25 )); then
```

Comment:
Shouldn't this have been VERSION?

Reply:
It should be, but this didn't cause the issue, since we didn't use 1.25 for the E2Es